concent_api/verifier/utils.py

```diff
@@ -36,7 +36,7 @@
 from .constants import UNPACK_CHUNK_SIZE
 
 
-sys.path.append('/golem/entrypoints/scripts/render_tools')
+sys.path.append(os.path.join(settings.BLENDER_RENDER_TOOLS_DIR, '/golem/entrypoints/scripts/render_tools'))
```

Comment:
I don't think utils is a good location for this. If I was looking for something like this I would not think to look here. Why not put it in some more obvious location, e.g. in the file with verifier's views or even somewhere in `concent_api/concent_api/`? Actually the `manage.py` file might be the best location though I'm not sure if you'll be able to import config there without any annoying side-effects.

Reply:
Ok, moved to `manage.py`.